src/theme/DocCardList/index.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import clsx from "clsx";
+import {
+  useCurrentSidebarCategory,
+  filterDocCardListItems,
+} from "@docusaurus/theme-common";
+import DocCard from "@theme/DocCard";
+function DocCardListForCurrentSidebarCategory({ className }) {
+  const category = useCurrentSidebarCategory();
+  return <DocCardList items={category.items} className={className} />;
+}
+export default function DocCardList(props) {
+  const { items, className } = props;
+  if (!items) {
+    return <DocCardListForCurrentSidebarCategory {...props} />;
+  }
+  const filteredItems = filterDocCardListItems(items);
+  return (
+    <section className={clsx("row", className)}>
+      {filteredItems.map((item, index) => (
+        <article key={index} className="col col--6 margin-bottom--lg">
+          <DocCard item={item} />
+        </article>
+      ))}
+    </section>
+  );
+}
```

Comment:
Blocking: 

If I remember correctly, there are no changes in this file. If that's the case, this file is unnecessary and we should delete it!

If I'm not remembering correctly, and there are customizations in here, we should capture that with a comment at the top, like I described in DocCard/index.js.

Reply:
Removed in my latest commit 👍